react-search-component/my-app/src/FilteredItems.js

```diff
@@ -0,0 +1,24 @@
+export default function FilteredItems(props) {
```

Comment:
It's usually easier to use destructuring to declare the props. That makes the code easier to decipher, especially when you want to use the component and can't remember what props to pass!